app/services/municipality-list.ts

```diff
@@ -54,11 +54,10 @@ export default class MunicipalityListService extends Service {
    * @returns a Promise for a joined string of those locations' id's, or undefined
    */
   async getLocationIdsFromLabels(
-    labels?: Array<string> | string,
-    stringSeperator = seperator
```

Comment:
We didn't pass custom separotors anywhere so I simply removed it. We could add an option to the serialize / deserialize utils to configure the separator, but I don't think it's needed.